src/options.rs

```diff
@@ -113,6 +113,11 @@ pub struct BuildOptions {
     #[structopt(long = "target-dir")]
     /// Target dir option to pass to cargo build.
     pub target_dir: Option<String>,
+
+    #[structopt(long = "coverage")]
+    /// Instrument program code with source-based code coverage information.
+    /// The profiler data will be saved in the given file.
+    pub coverage_output_file: Option<String>,
```

Comment:
It would be great if the average user didn't have to think about what to name the profile data file. Is there an existing convention that we can use by default?

Reply:
clang uses `default.profraw` if not told otherwise:

> When the program exits it will write a raw profile to the path specified by the LLVM_PROFILE_FILE environment variable. If that variable does not exist, the profile is written to default.profraw in the current directory of the program. 

https://clang.llvm.org/docs/SourceBasedCodeCoverage.html#running-the-instrumented-program

Reply:
I am now generating `default-<corpus file name>.profdata` files for each individual input file from the corpus, and merging them in the end using the profdata tool. The user doesn't need to provide names for the raw files.